package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "caps",
+  "version": "1.0.0",
+  "description": "caps",
+  "main": "caps.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
change to `"test": "jest --verbose --coverage"`, also if you want you can add `"start":"node caps.js"`